apps/spruce/src/pages/task/taskTabs/ExecutionTasksTable.tsx

```diff
@@ -117,42 +108,27 @@ export const ExecutionTasksTable: React.FC<Props> = ({
   );
 };
 
-const getInitialSorting = (queryParams: {
-  [key: string]: any;
-}): SortingState => {
-  const {
-    [TableQueryParams.SortBy]: sortBy,
-    [TableQueryParams.SortDir]: sortDir,
-    [TableQueryParams.Sorts]: sorts,
-  } = queryParams;
-
-  let initialSorting = [{ id: TaskSortCategory.Status, desc: false }];
-  if (sortBy && sortDir) {
-    initialSorting = [
-      {
-        id: sortBy as TaskSortCategory,
-        desc: sortDir === SortDirection.Desc,
-      },
-    ];
-  } else if (sorts) {
-    const parsedSorts = parseSortString<
-      "id",
-      "direction",
-      TaskSortCategory,
-      {
-        id: TaskSortCategory;
-        direction: SortDirection;
-      }
-    >(sorts, {
-      sortCategoryEnum: TaskSortCategory,
-      sortByKey: "id",
-      sortDirKey: "direction",
-    });
-    initialSorting = parsedSorts.map(({ direction, id }) => ({
-      id,
-      desc: direction === SortDirection.Desc,
-    }));
-  }
+const getInitialSorting = (sorts: string): SortingState => {
+  const initialSorting = sorts
+    ? parseSortString<
+        "id",
+        "direction",
+        TaskSortCategory,
+        {
+          id: TaskSortCategory;
+          direction: SortDirection;
+        }
+      >(sorts, {
+        sortCategoryEnum: TaskSortCategory,
+        sortByKey: "id",
+        sortDirKey: "direction",
+      }).map(({ direction, id }) => ({
+        id,
+        desc: direction === SortDirection.Desc,
+      }))
+    : [{ id: TaskSortCategory.Status, desc: false }];
 
   return initialSorting;
 };
+
+const defaultSortQueryParam = `${TaskSortCategory.Status}:${SortDirection.Asc}`;
```

Comment:
Can we construct this with toSortString to ensure its consistent with that logic?